skops/card/tests/test_card.py

```diff
@@ -114,6 +114,14 @@ def test_metadata_keys(destination_path, model_card):
         assert "tags: dummy" in f.read()
 
 
+def test_add_metrics(destination_path, model_card):
+    model_card.add_metrics(**{"acc": 0.1})
+    model_card.add_metrics(f1=0.1)
+    model_card.save(Path(destination_path) / "README.md")
+    with open(Path(destination_path) / "README.md", "r") as f:
+        assert "acc" and "f1" and "0.1" in f.read()
```

Comment:
This line doesn't do what it's supposed to do. `"acc" and "f1" and "0.1"` is evaluated to `"0.1"` and only the latter is checked. You need to check all individually.

Reply:
Thanks a lot for noting it, I put them separately.